drivers/ISOSR.py

```diff
@@ -2,13 +2,13 @@
 #
 # Copyright (C) Citrix Systems Inc.
 #
-# This program is free software; you can redistribute it and/or modify 
```

Comment:
We should probably avoid all the trailing whitespace changes in this commit so that it's clear what is actually changing.

Reply:
Sorry slighly over zealous. 
Will fix.

Reply:
Fixed